model/schema/variation.yaml

```diff
@@ -114,6 +112,16 @@ classes:
     slots:
       - assembly
       - chromosome
+      - exon_number
+      - intron_number
+      - variation_strand
+      - number_additional_dna_base_pairs
+      - number_removed_dna_base_pairs
+      - inserted_sequence
+      - deleted_sequence
```

Comment:
This can be inferred from `reference_sequence` and `variant_sequence` - do we need both pairs of slots?